pkg/ccl/backupccl/restore_span_covering.go

```diff
@@ -89,6 +89,8 @@ func makeSimpleImportSpans(
 	introducedSpanFrontier *spanUtils.Frontier,
 	lowWaterMark roachpb.Key,
 	targetSize int64,
+	checkpointingSpanFrontier *spanUtils.Frontier,
```

Comment:
@steven remarked that it might be cleaner to conduct span filtering after makeSimpleImportSpans on the constructed restoreSpanEntries. It would lead to a bit of wasted work in makeSimpleImportSpans (processing requiredSpans that have already finished), but steven's idea would make the code much easier to follow. 

So, wdyt of not changing makeSimpleImportSpans at all, and doing the filter after it?

Reply:
lol tagged the wrong @stevendanna initially